src/main/java/codeu/model/store/basic/TagStore.java

```diff
@@ -130,10 +130,10 @@ public void addTag(Tag tag) {
   }
 
    /** Check whether a Conversation title is already known to the application. */
-  public boolean isTagTake(Tag tag) {
+  public boolean isTagTaken(String tag) {
```

Comment:
Let's call this parameter `tagContent` as well, for consistency. Or having the parameter of `getTag` be `tag` instead of `tagContent`.